dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/utils/LogUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.dubbo.remoting.utils;
+
+import org.apache.dubbo.common.logger.Logger;
+import org.apache.dubbo.common.logger.LoggerFactory;
+import org.apache.dubbo.remoting.exchange.Request;
+import org.apache.dubbo.remoting.exchange.Response;
+
+public class LogUtils {
+    private static Logger logger = LoggerFactory.getLogger(LogUtils.class);
+
+    /**
+     * only log body in debugger mode for size & security consideration.
+     *
+     * @param message
+     * @return
+     */
+    public static Object getRequest(Object message) {
```

Comment:
Is there a better name to express this method？This method looks only in a scene with exception to  remove sensitive data。

Reply:
update to getReqeustWithoutData, please go ahead.